seed/csharp-sdk/accept-header/src/SeedAccept/Core/RawClient.cs

```diff
@@ -569,6 +579,53 @@ and not string
         return url;
     }
 
+    private static List<KeyValuePair<string, string>> GetQueryParameters(BaseApiRequest request)
+    {
+        var result = TransformToKeyValuePairs(request.Query);
+        if (
+            request.Options?.AdditionalQueryParameters != null
```

Comment:
if `request.Options?.AdditionalQueryParameters` is null, I'd early return.
Also, the `is null` and `is not null` is preferred

Reply:
Great call, definitely in favor of the early return.